### DISCUSSION TITLE
Lining up the laser in the correct starting point

### DISCUSSION THREAD

**Discussion by JerseyHoney67** [arkypita/LaserGRBL#1431] [Q&A]
How do you find the correct staring point of the laser? I know how to get the laser to turn on and you can see the light but how can find out you have the picture exactly where you want it to be?

**Comment by arkypita** [comment]
Custom buttons?

https://lasergrbl.com/usage/custom-buttons/

**Reply by JerseyHoney67** [reply]
Hi what I was looking for was an easy step by step on how to prepare and execute the laser showing exactly where the image will be created

**Reply by JerseyHoney67** [reply]
Having the buttons is great but a complete understanding of how they operate would be very useful.

**Comment by StuartB4** [comment]
If you have the same Frame button I have the code is like this.

**G0 X[left] Y[bottom]**   (This makes sure the laser is at 0,0 position of the image on the workpiece, bottom left. )
**M3 S5 F1000**             (M3 is Laser On - S5 is the laser Power - F1000 is the frame speed.)
**G1 Y[top]**                   (This moves the laser up to the top left, distance is determined by the image size you enter when preparing it.) 
**G1 X[right]**                 (This moves the laser across to the top right.)
**G1 Y[bottom]**             (This moves the laser down to the bottom right.)
**G1 X[left]**                    (This moves the laser across to the bottom left.)
**M5**                             (M5 switches laser off.)
**G0 X0 Y0**                    (This sends the laser to the home position. )

**S** setting can be adjusted to increase or decrease laser power. You only need it set high enough to just come on and be visible.
**F** setting can be adjusted to increase or decrease framing speed.

**Comment by JerseyHoney67** [comment]
Thanks Stuart



Warm Regards


*Shaun Gell*

*Managing Director*

+44 7797 774488 www.jerseyhoney.com <http://jersey-honey.com/>

**Comment by alastaira** [comment]
One thing a lot of people don't seem to realise when it comes to lining up jobs is that you can double click in the LaserGRBL preview window to automatically jog the laser to that point, which you can use to check exactly where certain features will be burned (e.g. a particular detail in a geometric design, the eyes of the subject, a letter of text etc.). Then you can line up your material with that exact set of points before returning home and starting a burn.
